_posts/en/posts/2024-11-05-cb-stall-hindering-propagation.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Disclosure of hindered block propagation due to stalling peers
+name: blog-disclose-stalling-peers-hindering-propagation
+id: en-blog-disclose-stalling-peers-hindering-propagation
+lang: en
+type: advisory
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security announcement. English posts only
+announcement: 1
+
+excerpt: >
+    A peer could hinder block propagation by announcing blocks first and then simply withholding the block.
+---
+
+Before Bitcoin Core v25.1, an attacker can cause a node to not
+download the latest block.
+
+This issue is considered **Medium** severity.
+
+## Details
+
+When receiving a new block announcement via a headers or compact
+blocks message, the delivering peer is requested either the full
+block or missing transaction details by the receiving node. If
+the announcing peer then doesn't respond as the peer to peer
+protocol requires, the affected Bitcoin Core node will wait
+up to 10 minutes before disconnecting the peer and another
+block download attempt is made. If the attacker is able to
+make multiple incoming or outgoing connections, this process
+can be repeated.
+
+This can cause network degradation with network convergence
+slowdown, making mining payouts less fair, and can cause
+liveliness issues.
+
+This issue was further exacerbated by other issues [disclosed recently](https://bitcoincore.org/en/2024/10/08/disclose-large-inv-to-send/), when mempools were relatively heterogeneous, disallowing
```

Comment:
```suggestion
This issue was exacerbated if it occurred in conjunction with the recently disclosed [inventory build-up](https://bitcoincore.org/en/2024/10/08/disclose-large-inv-to-send/), when mempools were relatively heterogeneous, disallowing
```

Reply:
fwiw it's not just that issue, per se, but any issue that results in unsynced mempools.

Reply:
Went with:
```diff
diff --git a/_posts/en/posts/2024-11-05-cb-stall-hindering-propagation.md b/_posts/en/posts/2024-11-05-cb-stall-hindering-propagation.md
index 0323339..e144662 100644
--- a/_posts/en/posts/2024-11-05-cb-stall-hindering-propagation.md
+++ b/_posts/en/posts/2024-11-05-cb-stall-hindering-propagation.md
@@ -36,7 +36,9 @@ can be repeated.
 Delaying block delivery can cause network degradation by slowing down network convergence,
 making mining payouts less fair, and causing liveliness issues.
 
-This issue was further exacerbated by other issues [disclosed recently](https://bitcoincore.org/en/2024/10/08/disclose-large-inv-to-send/), when mempools were relatively heterogeneous, disallowing
+This issue was further exacerbated by other issues disclosed recently (for
+instance the [inventory build-up](https://bitcoincore.org/en/2024/10/08/disclose-large-inv-to-send/)),
+when mempools were relatively heterogeneous, disallowing
 opportunistic reconstruction of compact blocks by honest peers.
 
 A mitigation was introduced in [#27626](https://github.com/bitcoin/bitcoin/pull/27626),
```